binder/Desktop/Spyder.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Spyder (Development Version)
+Exec=/bin/bash -c "exec ${HOME}/bootstrap.py"
+# FIXME: Don't hardcode Path?
+Path=/home/jovyan/Desktop
```

Comment:
Could we add an icon? It would make it easier for easier for users to know what to click

Reply:
We could use this one.

https://github.com/spyder-ide/spyder/blob/master/img_src/spyder.ico

Reply:
Done